Makefile

```diff
@@ -17,9 +16,12 @@ endif
 .PHONY: help
 help:  ## This.
 	@perl -ne 'print if /^[a-zA-Z_-]+:.*## .*$$/' $(MAKEFILE_LIST) \
-	| sort \
 	| awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: runserver
```

Comment:
now called `runserver` instead of `run`.

Reply:
thanks